image_proc/include/image_proc/rectify.hpp

```diff
@@ -56,6 +56,7 @@ class RectifyNode
   image_transport::CameraSubscriber sub_camera_;
 
   int queue_size_;
+  bool use_qos_;
```

Comment:
Is there a reason this was implemented as a member variable in `rectify` but a temporary in `debayer`?

Reply:
Apologies for taking a minute on this, but it was due to it being needed in two separate methods, the constructor and subscribeToCamera. It could be instead be a local value and retrieved multiple times, or passed into the subscribeToCamera method if those seem like better paradigms.

Reply:
I get it now. Thanks for the clarification. It's fine.